.storybook/preview-head.html

```diff
@@ -7,5 +7,9 @@
     --mb-color-focus: #cbe2f7;
     --mb-color-bg-dark: #93a1ab;
     --mb-color-bg-light: #f9fbfc;
+    --mb-color-bg-white: #fff;
+    --mb-color-text-white: #fff;
+    --mb-color-success: #84bb4c;
+    --mb-color-summarize: #88bf4d;
```

Comment:
why doesn't `warning` get the same css custom property as other colors? I see it's only used in a handful of places, but it's still a semantic color.